libs/libcommon/src/libcommon/prometheus.py

```diff
@@ -76,15 +100,35 @@ def update_responses_in_cache_total() -> None:
         ).set(cache_metric.total)
 
 
-def update_assets_disk_usage(assets_directory: StrPath) -> None:
+def update_disk_gauge(gauge: Gauge, directory: StrPath) -> None:
     # TODO: move to metrics, as for the other metrics (queue, cache)
-    total, used, free, percent = disk_usage(str(assets_directory))
+    total, used, free, percent = disk_usage(str(directory))
```

Comment:
In the next line, it should update the labels for the new gauge var, not for ASSETS_DISK_USAGE, I think this is the problem.